L1Trigger/TrackTrigger/interface/TrackQuality.h

```diff
@@ -0,0 +1,75 @@
+/*
+Track Quality Header file
+
+C.Brown 28/07/20
+*/
+
+#ifndef L1Trigger_TrackTrigger_interface_TrackQuality_h
+#define L1Trigger_TrackTrigger_interface_TrackQuality_h
+
+#include <iostream>
+#include <set>
+#include <vector>
+#include <memory>
+#include <string>
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h"
```

Comment:
this include doesn't appear to be necessary in this header file (maybe it should be moved to the .cc)

Reply:
done